.github/workflows/doc-lint-test.yaml

```diff
@@ -231,7 +282,29 @@ jobs:
       - name: Install geoips documentation packages
         run: pip install -e .[doc]
 
+      - name: Build release notes from yaml files
+        id: release-note-generator
+        run: |
+          update_this_file="$(cat $BASE_GEOIPS_PATH/update_this_release_note)"
+          release_notes_dir="$(dirname $update_this_file)/$(basename $update_this_file .rst)"
+          release_version="$(basename $update_this_file .rst)"
+          echo "Running brassy on directory: $release_notes_dir"
+          brassy --release-version $release_version --no-rich --output-file $update_this_file $release_notes_dir
+          echo "release_note_file=$update_this_file" >> $GITHUB_OUTPUT
+          cat $update_this_file
+      - name: Pinken release note
```

Comment:
Given how this works, I wonder if `update_this_release_note` should just be called `next_expected_release` and contain only the version number. It seems easier all around. If we want to do that, though, it can be done in a new issue and PR.

Thoughts?